src/compiler/checker.ts

```diff
@@ -920,6 +921,7 @@ namespace ts {
             nameNotFoundMessage: DiagnosticMessage,
             nameArg: __String | Identifier,
             isUse: boolean,
+            includeGlobals: boolean,

```

Comment:
seems wierd to have to opt-in resolving a global. i would expect this be the opposite, that you opt out-in to skip globals